### DISCUSSION TITLE
Password protected rides

### DISCUSSION THREAD

**Discussion by ghost** [OpenRCT2/OpenRCT2#15703] [Feature requests]
Hello, I see people deleting rides in my server without permission which is griefering and I had an astonishing idea to add to OpenRCT2 which is rides being password protected to prevent grieferings. Does anyone agree with me?

**Comment by Margen67** [comment]
I think having permissions assignable to rides would be better.

**Comment by IntelOrca** [comment]
This would then allow griefers to block other players' construction with their own rides.

**Comment by Jeroen-N** [comment]
Would it be possible to assign plots of land from your park to specific people to build on? I agree having passwords on rides would making it even worse.

**Comment by starfrost013** [comment]
A limit on the zone size would be needed. Someone could zone the entire park for them and therefore make it unbuildable to anyone else.

**Comment by Margen67** [comment]
Then make it require a permission. Don't give it to griefers.

**Comment by fromthere** [comment]
This should require openrct2 own save file format. there's no way to save to sv6 without breaking format.
